services/brig/src/Brig/API/Internal.hs

```diff
@@ -171,6 +176,34 @@ authAPI =
     :<|> Named @"login-code" getLoginCode
     :<|> Named @"reauthenticate" reauthenticate
 
+federationRemotesAPI :: ServerT BrigIRoutes.FederationRemotesAPI (Handler r)
+federationRemotesAPI =
+  Named @"get-federation-remotes" getFederationRemotes -- TODO: get this from TVar!
```

Comment:
this TODO is important!

Reply:
Shouldn't getting the federation remotes be reading from the DB directly in this case? Reading from a TVar/IORef will be introducing another polling delay for services that depend on this API to get the federation domain state.

Reply:
167083a63d376859cfd67e3b076d88a4ed90f3db  :+1: